YearInReview/Model/Reports/ReportGenerator.cs

```diff
@@ -55,8 +58,26 @@ public async Task<Report1970> Generate(int year)
 			var games = _playniteApi.Database.Games;
 			var activities = await _gameActivityExtension.GetActivityForGames(games);
 			var filteredActivities = _specificYearActivityFilter.GetActivityForYear(year, activities);
+			
+			ShowReportGeneratedNotification(year);
```

Comment:
Probably move `_composer1970.Compose(year, filteredActivities)` to a variable, and after the variable is created call `ShowReportGeneratedNotification(year);`

Reason? If composing a new report would fail with an exception, we would still show a notification, that we generated it. Better show it only after we have it generated.